foods.html

```diff
@@ -11,13 +11,16 @@
   <h1>Manage Foods</h1>
   <div class="manage-foods">
     <form class="new-food">
+      <h5>Add New Food</h5>
       <input type="text" name="name" placeholder="Name" id="new-name"></br>
       <div id="new-name-error">Please enter a name!</div>
       <input type="number" name="calories" placeholder="Calories" id="new-calories">
       <div id="new-calories-error">Please enter a calorie count!</div>      
       <button type="submit" id="submit">Submit</button>
     </form>
 
+    <input type="text" id="myInput" placeholder="Search for foods..">
```

Comment:
I think the story may say "Filter by Name"

Reply:
Got it- I made the adjustment! Thanks for checking that.